io.github.mpc_qt.Mpc-Qt.yml

```diff
@@ -0,0 +1,167 @@
+            sources:
+              - type: archive
+                url: 'https://linuxtv.org/downloads/v4l-utils/v4l-utils-1.22.1.tar.bz2'
+                sha256: '65c6fbe830a44ca105c443b027182c1b2c9053a91d1e72ad849dfab388b94e31'
+
+          - name: nv-codec-headers
+            cleanup:
+              - '*'
+            no-autogen: true
+            make-install-args:
+              - PREFIX=/app
+            sources:
+              - type: archive
+                url: 'https://github.com/FFmpeg/nv-codec-headers/releases/download/n11.1.5.1/nv-codec-headers-11.1.5.1.tar.gz'
+                sha256: 'a28cdde3ac0e9e02c2dde7a1b4de5333b4ac6148a8332ca712da243a3361a0d9'
+
+          - name: ffmpeg
+            cleanup:
+              - /include
+              - /lib/pkgconfig
+              - /share/ffmpeg/examples
+            config-opts:
+              - '--enable-shared'
+              - '--disable-static'
+              - '--enable-gnutls'
+              - '--enable-gpl'
+              - '--disable-doc'
+              - '--disable-programs'
+              - '--disable-encoders'
+              - '--disable-muxers'
+              - '--enable-encoder=png,libwebp'
+              - '--enable-libv4l2'
+              - '--enable-libdav1d'
+              - '--enable-libfontconfig'
+              - '--enable-libfreetype'
+              - '--enable-libopus'
+              - '--enable-librsvg'
+              - '--enable-libvpx'
+              - '--enable-libmp3lame'
+              - '--enable-libwebp'
+            sources:
+              - type: archive
+                url: 'https://ffmpeg.org/releases/ffmpeg-4.4.1.tar.gz'
+                sha256: '82c98f74777f623710b72f9a3389fd38c1ed93bc661107e65df19234e395f6b9'
+
+          - name: libass
+            cleanup:
+              - /include
+              - /lib/*.la
+              - /lib/pkgconfig
+            config-opts:
+              - '--disable-static'
+            sources:
+              - type: archive
+                url: 'https://github.com/libass/libass/releases/download/0.15.2/libass-0.15.2.tar.gz'
+                sha256: '1b2a54dda819ef84fa2dee3069cf99748a886363d2adb630fde87fe046e2d1d5'
+
+          - name: uchardet
+            buildsystem: cmake-ninja
+            config-opts:
+              - '-DCMAKE_BUILD_TYPE=Release'
+              - '-DBUILD_STATIC=0'
+            cleanup:
+              - /bin
+              - /include
+              - /lib/pkgconfig
+              - /share/man
+            sources:
+              - type: archive
+                url: 'https://gitlab.freedesktop.org/uchardet/uchardet/-/archive/v0.0.7/uchardet-v0.0.7.tar.gz'
+                sha256: 'f3635d1d10e1470452bc42c1bf509451a9926b399a11740a9949e86069d69f58'
+
+      - name: yt-dlp
+        no-autogen: true
+        no-make-install: true
+        make-args:
+          - yt-dlp
+          - PYTHON=/usr/bin/python3
+        post-install:
+          - install yt-dlp /app/bin
```

Comment:
destination is a file name. So this might not do what you want.

Reply:
I tested a youtube stream and it works the way it is.